mw-webapp/src/service/auth/handleUserAuthState.ts

```diff
@@ -18,8 +20,10 @@ export const handleUserAuthState = (setUser: React.Dispatch<React.SetStateAction
     const userDocSnapshot = await getDoc(userDocRef);
 
     if (!userDocSnapshot.exists()) {
-      writeNewUserCredentials(currentUser);
+      await writeNewUserCredentials(currentUser);
```

Comment:
could you check why do we need to rewrite something on each login? Looks strange

Reply:
We talk about.